package.json

```diff
@@ -4,17 +4,20 @@
   "description": "The main API for the texvet.io application.",
   "main": "",
   "scripts": {
-    "build": "sls package --stage development",
     "build:demo": "sls package --stage demo",
-    "build:prod": "sls package --stage production",
+    "build:development": "sls package --stage development",
+    "build:production": "sls package --stage production",
     "build:staging": "sls package --stage staging",
+    "compose": "docker-compose -f ./database/docker-compose.yml up -d ",
```

Comment:
The AWS_PROFILE environment variable won't work in CircleCI. The seeder needs to use the configuration in the .env.yml file.

Reply:
done

Reply:
We still need to remove the AWS_PROFILE stuff here